galeracluster/source/documentation/mysql-wsrep-options.rst

```diff
@@ -1984,15 +1984,15 @@ When the node makes a state transfer request it calls on an external shell scrip
 
 Galera Cluster ships with a number of default scripts that the node can use in state snapshot transfers. The supported methods are:
 
-- ``mysqldump`` This is slow, except for small data-sets, but is the most tested option.
+- ``mysqldump`` This is slow, except for small data-sets, but is the most tested option. On MariaDB Server, available from version 10.11.
```

Comment:
mysqldump has been available for a very long time - its slower, yes. BUt not 10.11.